docs/tutorials/adding-new-project.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Adding a new project
+subject: Projects
+---
+
+We currently have a [projects page] where you can browse projects that have
+been built using ev3dev. You can add your own too!  Each project gets a
+dedicated page for the author to explain what they have been working on, as
+well as provide videos, pictures, build instructions, code, and any other media
+that pertains to the project. This is where we explain how you can submit your
+project page to be hosted on the site.
+
+##Overview
+
+All the projects on the website are stored in the [projects folder].
+Each post is saved as a Markdown file, and is automagically converted to HTML
+when you publish your changes (read more about Markdown
+[here][markdown basics]). To submit a page,
+all you need to do is add a markdown file in that folder and submit a pull
+request. We recommend that you start with a copy of the [template project].
+You can see the example file live on [the website][template page].
+
+This guide will walk you through submitting a page without the need for any git
+experience or local tools installed on your computer. All you need is a
+browser.
+
+## Step 1: Making a Copy of the Website Source
+
+The first step to submitting your project is forking the source code on GitHub.
+To do this, open a browser and navigate to the [website repository]. Make
+sure that you are signed in to your GitHub account, and then click the "fork"
+button in the upper-right corner.
+
+<img src="https://github-images.s3.amazonaws.com/help/repository/fork_button.jpg" style="max-width: 300px;" />
+
+It should take a second, but when GitHub finishes processing your request you
+should be at a page very similar to where you started, except now you are
+looking at your own copy.
+
+##Step 2: Setting Up A Preview (Optional)
+
+If you want to be able to preview your new page before you submit it, and are
+willing to add a bit of extra complexity, you have to create a new branch
+called `gh-pages`. Adding this branch will let you preview the whole site, so
+you can see how your page will look when it is published (which is different
+from the markdown preview that you will see on GitHub). To do this, click the
+box where it says "branch: master". It should expand to show you a text box.
+Type the text `gh-pages` in to the box that appears (make sure that it is
+exactly that text; this is important). It should give you the option to create
+a new branch; click that button.
+
+<img src="/images/Website/Branching-on-GitHub.png" style="max-width: 50%;" />
```

Comment:
We have a include template for screenshots.

    {% include screenshot.html source="/images/Website/Branching-on-GitHub.png" scale="50" %}